mix.exs

```diff
@@ -19,6 +20,18 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [contributors: ["José Valim", "Milton Mazzarri", "Friedel Ziegelmayer",
+                   "eksperimental", "John Warwick", "Rafael Mendonça França",
+                   "Alexei Sholik", "Eric Meadows-Jönsson", "Peter Minten", "Sergey Nartimov",
+                   "Samuel Tonini", "Joseph Wilk", "Dave Thomas", "Tony Han",
+                   "Josh Adams", "Andrew Forward", "Brian Gianforcaro", "Kevin Montuori",
+                   "Lau Taarnskov", "Pedro Medeiros", "rightfold", "Stefan", "Vicent Marti",
+                   "Yuki Ito", "Chris Hyndman", "Thomas", "lidashuang"],
```

Comment:
Here are my two cents.
I think putting the whole list is a bit too much, I think there should be a threshold, maybe 10 commits, 15 commits. This list will be pollutated with a lot of information that doesn'tmean anything, like the name of someone whoe submitted a PR fixing a typo.
on the other hand, i'm ok with not including the list of contributors.
Regarding @josevalim's idea of only putting the current team, I think it would not not do justice for people who contributed substantial chunks of code in the past. If we can include a field called :maintainers, that will fairer.